### ISSUE TITLE
there is a warning in the function "_select_logits_with_mask"

### ISSUE THREAD

**Issue by fangd123** [airaria/TextBrewer#7]
Warning: masked_scatter_ received a mask with dtype torch.uint8, this behavior is now deprecated,please 
use a mask with dtype torch.bool instead.

when I change `mask = mask.unsqueeze(-1).expand_as(logits).to(torch.uint8)` with `mask = mask.unsqueeze(-1).expand_as(logits).to(torch.bool)` the warning disappear.

my environment is 
```
python 3.7
pytorch 1.4
cuda 10.0
cudnn7.6
```

**Comment by airaria** [comment]
Thank you for point it out. I'll fix it.

**Comment by airaria** [comment]
Fixed in dd1ecbb
